pkgs/by-name/md/mdbook-pdf-outline/package.nix

```diff
@@ -26,7 +26,7 @@ python3Packages.buildPythonApplication rec {
   meta = with lib; {
     homepage = "https://github.com/HollowMan6/mdbook-pdf";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ nix-julia ];
+    maintainers = with maintainers; [ hollowman6 ];
```

Comment:
Non-blocking:

I prefer splitting maintainer changes into its own commit for clarity, as such changes are related to the merge-bot permissions.

Reply:
Now this is a separate commit for this PR.